src/index.html

```diff
@@ -15,8 +15,73 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          class="imagem"
+          src="./images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+      <nav class="links">
+        <a
+          href="#Apple"
+          class="is-active link"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. The `href` for 'Apple' should be '#apple' instead of '#Apple'. URLs are case-sensitive and traditionally lowercase.

Reply:
justo